packages/mongo-nav/src/org-nav/OrgNav.tsx

```diff
@@ -241,6 +265,7 @@ export default function OrgNav({
             className={cx(rightSideLinkStyle, linkText, {
               [activeLink]: activeNav === 'admin',
             })}
+            data-testid="admin-link"
```

Comment:
What are the chances this test ID might collide with another on the page?

Reply:
On Cloud? None, because we don't use Jest, plus our Admin app is partially Java Static Pages.

On this component? Likely none as well, unless you see another potential usage in the design that I missed.

In general, if the testid needs to be read outside of the component, it's the user's responsibility for selecting which one (ex: if there are many usages of button).